src/css/custom.css

```diff
@@ -21,8 +21,19 @@
   h2 {
     font-size: 1.5rem;
   }
+
+  .navbar {
+    background: rgba(255, 179, 212, 0.8);
```

Comment:
The alpha value is a little high. It would be better with a lower value, such as 0.5.

![Light Mode](https://github.com/user-attachments/assets/58fad11e-aac2-4d3e-8af3-0ad46ad53547)

![Dark Mode](https://github.com/user-attachments/assets/adcff2b4-677d-4961-9caa-4286434cee13)

Reply:
I have tested and I beieve that setting alpha value as 0.5 seems a little bit visual-distracting, setting it to 0.6 instead.